src/router/schema.ts

```diff
@@ -56,7 +56,10 @@ export async function push(req, res) {
 		})
 	);
 
-	const data = await pushAndValidateSchema({ service });
+	const data = await pushAndValidateSchema({
+		service,
+		forcePush: req.headers['force-push'],
```

Comment:
what is this needed for?

Reply:
This is not in the code anymore 🙏 

I'm going to close this MR, I thought we already did it. In the future we are going to prepare better PRs, with some documentation, so it's easier for you